packages/client/src/app/shared/global/global.state.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { State, Action, Selector, StateContext } from '@ngxs/store';
+import { from, Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { GlobalService } from './services';
+import { GlobalStateModel } from './global-state.model';
+import { Global } from './global.actions';
+
+import { ContentFilter } from '@pulp-fiction/models/content';
+
+@State<GlobalStateModel>({
+    name: 'global',
+    defaults: {
+        filter: ContentFilter.Default
+    }
+})
+@Injectable()
+export class GlobalState {
```

Comment:
Would a better name for this stuff be something like "UserPreferences", or maybe even "VisitorPreferences" to distinguish that this is for _all_ visitors to the site, instead of just logged-in ones?